modules/iam-role-for-service-accounts-eks/policies.tf

```diff
@@ -1,3 +1,41 @@
+################################################################################
+# API Gateway Controller Policy
+################################################################################
+
+data "aws_iam_policy_document" "api_gateway_controller" {
+  count = var.create_role && var.attach_api_gateway_controller_policy ? 1 : 0
+
+  # https://github.com/aws/aws-application-networking-k8s/blob/main/examples/recommended-inline-policy.json
```

Comment:
In the comment, does it make sense to point to a tag/release instead of a moving `main` branch?

The rest LGTM.

Reply:
good point - corrected in https://github.com/terraform-aws-modules/terraform-aws-iam/pull/378/commits/5ca15c6b85f6802b384d6ab9687d136c11ac7a5c